e2e/rook-test-framework/enums/rook_platform_types.go

```diff
@@ -0,0 +1,41 @@
+package enums
+
+import (
+	"errors"
+	"strings"
+)
+
+type RookPlatformType int
+
+const (
+	Kubernetes RookPlatformType = iota + 1
+	BareMetal
+	StandAlone
+	None
+)
+
+var platforms = [...]string{
+	"Kubernetes",
+	"BareMetal",
+	"StandAlone",
+	"None",
+}
+
+func (platform RookPlatformType) String() string {
+	return platforms[platform-1]
+}
+
+func GetRookPlatFormTypeFromString(name string) (RookPlatformType, error) {
```

Comment:
in golang I personally find enums not very useful. it's easier to just compare against the string than to translate it to the type, which is just really an int. It removes the need for helper methods like this and you could likely remove the whole `enums` package